<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductFormDialog</class>
 <widget class="QDialog" name="ProductFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 54, 59);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI Variable</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Definir información del producto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="producCategory_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Categoría: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="productCategory_comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">  QComboBox {
    border: 2px solid rgb(53, 54, 59);
    border-radius: 15px;
    color: rgb(255, 255, 255);
    background-color: rgb(38, 39, 44);
    padding-left: 20px;
    padding-right: 20px;
  }

  /* Estilo para el área desplegable */
QComboBox QAbstractItemView {
    border: 2px solid rgb(53, 54, 59);
    background-color: rgb(38, 39, 44);
    color: rgb(255, 255, 255);
    selection-background-color: rgb(63, 64, 70);
    selection-color: white;
 }

QComboBox::drop-down {
    width: 0px; /* Hacemos que la flecha no ocupe espacio */
    border: none;
    background-color: transparent;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="productName_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="productName_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
  border: 2px solid rgb(53, 54, 59);
  border-radius: 15px;

  color: rgb(255, 255, 255);
  background-color: rgb(38, 39, 44);
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;
}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Para la siguiente información, porfavor introduzca los datos de los ingredientes de la siguiente manera: [Ingrediente] [Cantidad], [Ingrediente [Cantidad].  El sistema no tiene una unidad de medida, la unidad de medida se interpreta a partir de la unidad de medida utilizada para definir los productos de materia prima.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="productIngredients_label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Ingredientes: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="productIngredients_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  border: 2px solid rgb(53, 54, 59);
  border-radius: 15px;

  color: rgb(255, 255, 255);
  background-color: rgb(38, 39, 44);
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="productPrice_label">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Precio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="productPrice_doubleSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QDoubleSpinBox {
  border: 2px solid rgb(53, 54, 59);
  border-radius: 15px;

  color: rgb(255, 255, 255);
  background-color: rgb(38, 39, 44);
  padding-left: 20px;
  padding-right: 20px;
}

QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
  width: 0px; /* Hacer que los botones no sean visibles */
}
</string>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="productImageSelector_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 5px;
	padding: 10px;
	background-color: rgb(0, 153, 73);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(0, 181, 84);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(0, 125, 58);     /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
          </property>
          <property name="text">
           <string>Seleccionar imagen de producto</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="producImage_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Introduzca una imagen para previsualizarla.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptProduct_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 20px;
	padding-right: 20px px;
	background-color: rgb(0, 153, 73);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(0, 181, 84);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(0, 125, 58);     /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelProduct_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 20px;
	padding-right: 20px px;
	background-color: rgb(232, 54, 18);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(253, 59, 20);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(208, 48, 16);    /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
